components/expenses/Order.js

```diff
@@ -15,7 +15,7 @@ import { Flex } from '../Grid';
 import Link from '../Link';
 import LinkCollective from '../LinkCollective';
 import Moment from '../Moment';
-import StyledButton from '../StyledButton';
+import SmallButton from '../SmallButton';
```

Comment:
As described in https://github.com/opencollective/opencollective/issues/3223, we want to get rid of bootstrap, so each one of these `Button`/`SmallButton` removed is a win. Here it's a bit sad to re-introduce one. Would have been great to rather convert the other button to `StyledButton`, but as @znarf said this component is going to be removed soon so we shouldn't spend too much energy on it.

Reply:
Yeah I know we're not supposed to be using them but I think it was easier to match the `MarkAsPaidBtn` which is a `SmallButton` than spending time setting the height and width of the normal `Button` to try and match. Or I can make them both `StyledButton`?

Reply:
> Or I can make them both StyledButton?

As long as it's not too complex, that would be the best option. Otherwise we can go with this solution.